tests/baselines/reference/namedTupleMembersErrors.errors.txt

```diff
@@ -0,0 +1,27 @@
+tests/cases/conformance/types/tuple/named/namedTupleMembersErrors.ts(1,41): error TS5084: Tuple members must all have names or not have names.
+tests/cases/conformance/types/tuple/named/namedTupleMembersErrors.ts(3,32): error TS5084: Tuple members must all have names or not have names.
+tests/cases/conformance/types/tuple/named/namedTupleMembersErrors.ts(5,32): error TS5084: Tuple members must all have names or not have names.
+tests/cases/conformance/types/tuple/named/namedTupleMembersErrors.ts(7,20): error TS5085: Tuple members express optionality with a trailing question mark on the type, a question mark on the member name is invalid.
+tests/cases/conformance/types/tuple/named/namedTupleMembersErrors.ts(9,41): error TS5086: Rest tuple members are declared with a `...` on the type. A `...` on the name is invalid.
+
+
+==== tests/cases/conformance/types/tuple/named/namedTupleMembersErrors.ts (5 errors) ====
+    export type Segment1 = [length: number, number]; // partially named, disallowed
+                                            ~~~~~~
+!!! error TS5084: Tuple members must all have names or not have names.

```

Comment:
@sandersn @DanielRosenwasser 🚲 🏚️ - I rewrote this message like 3 times and they all sound awkward. I want to say something like `Tuple member namedness must be homogeneous` but that's maybe also bad.

Reply:
```
Tuple members must all be named or anonymous.
```

or 

```
All members of a tuple must be named or anonymous.
```

or

```
Named tuple members cannot be mixed with anonymous tuple members.
```

Reply:
But you might have to play where you issue the error depending on which one you pick. Related errors can help too as a learning device for what an "anonymous" member is.

Reply:
"Either all or no tuple member can be named"

or

"Naming of tuples is only supported if all members are named."

or

"Either all or no tuple members need to be named"